android/settings.gradle

```diff
@@ -7,7 +7,7 @@ include ':react-native-maps'
 project(':react-native-maps').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-maps/android')
 include ':realm'
 project(':realm').projectDir = new File(rootProject.projectDir, '../node_modules/realm/android')
-include ':RNBackgroundGeolocation', ':app'
-project(':RNBackgroundGeolocation').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-background-geolocation-android/android')
+include ':react-native-background-geolocation'
```

Comment:
In this PR, Android has support for a new version of background geolocation plugin, however iOS below this same library is missing.  Why this inconsistency?